### ISSUE TITLE
Possible to use only gif section? Error index with tabs.

### ISSUE THREAD

**Issue by tiltmaster** [Flyclops/tenor_flutter#9]
Hello!

So, the title pretty much says it, its amazing, but i just want to show the gif section/tab, doing the following:

```
tabs: [TenorTab(
                 name: 'GIFs',
                 view: TenorViewGifs(
                 client: tenorClient,
                 ),
          ), ],
```
causes a controller index issue, its expecting at least 2 so, i just want to show gifs, that would be nice.

**Comment by LegendAF** [comment]
I'll look into it, makes sense. Would you want it to still have the single tab at the top that says GIFs?

**Comment by tiltmaster** [comment]
> I'll look into it, makes sense. Would you want it to still have the single tab at the top that says GIFs?

Well it could be some optional thing, personally it shouldnt since its just gonna show GIFs, a customizable textfield and the same section would just be perfect for any usage.

**Comment by LegendAF** [comment]
@tiltmaster I agree that it shouldn't show. I'm not going to spend a ton of time showing it now but if someone finds this issue and truly wants the support I'll work on adding it then.

This should fix your issue
https://github.com/Flyclops/tenor_flutter/pull/10

**Comment by tiltmaster** [comment]
much appreciate @LegendAF
